about_me.html

```diff
@@ -0,0 +1,36 @@
+<html>
+    <head>
+        <title>About Me</title>
+        <link rel="stylesheet" href="about_me-style.css?Time=<?php echo microtime()?>" />
+
+    </head>
+
+    <body>
+        <h1>About Me</h1>
+        <p>I'm 20 years old. I'm from Chicago, but I'm currently living in St. Louis while I go to school and work.</p>
+        <div>
+        <h2>School</h2>
```

Comment:
Since this is inside a div, it should be indented. Although it doesn't look like the div is doing anything, so you might just remove it.